docker-compose.yml

```diff
@@ -1,18 +1,35 @@
 name: osaa-mvp
+
+x-common-config: &common-config
+  build: .
+  env_file: .env
+  volumes:
+    - ./datalake:/app/datalake
+    - ./sqlMesh:/app/sqlMesh
+  environment:
+    - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+    - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+    - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}
+    - S3_BUCKET_NAME=${S3_BUCKET_NAME}
+    - TARGET=${TARGET}
+    - USERNAME=${USERNAME}
+    - DB_PATH=/app/sqlMesh/osaa_mvp.db
+
 services:
   pipeline:
-    build: .
+    <<: *common-config
     container_name: osaa-mvp-main
-    env_file: .env
-    volumes:
-      - ./datalake:/app/datalake
-      - ./sqlMesh:/app/sqlMesh
     environment:
-      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
-      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
-      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}
-      - S3_BUCKET_NAME=${S3_BUCKET_NAME}
-      - TARGET=${TARGET}
-      - USERNAME=${USERNAME}
-      - DB_PATH=/app/sqlMesh/osaa_mvp.db
-    command: etl
+      - GATEWAY=${GATEWAY:-local}
+    profiles:
+      - default
+
+  ui:
+    <<: *common-config
+    container_name: osaa-mvp-ui
+    entrypoint: ["sqlmesh"]
+    command: ["ui", "--port", "${UI_PORT:-8000}"]
```

Comment:
in the `.env.example` it uses `8080`, should this be the same for consistency/simplicity?

Reply:
thanks Lisa, I've change it to be consistent with env.example